README.md

```diff
@@ -14,7 +14,7 @@ sudo SNIFFER_CONFIG=./configuration/SnifferConfigurationFile.json ./sniffer
 
 ## Limitations:
 1. This feature is using EBPF technology that is implemented only on linux.
-2. the linux kernel version that supported it 4.14
+2. the linux kernel version that supported it 5.4 and above.
```

Comment:
kernel version need to be 5.10

Reply:
https://github.com/inspektor-gadget/inspektor-gadget#kernel-requirements

Reply:
https://github.com/inspektor-gadget/inspektor-gadget/blob/main/docs/requirements.md